twoliter/src/cmd/build.rs

```diff
@@ -23,12 +37,102 @@ pub(crate) struct BuildVariant {
     project_path: Option<PathBuf>,
 
     /// The architecture to build for.
-    #[clap(long = "arch", default_value = "x86_64")]
+    #[clap(long = "arch", env = "BUILDSYS_ARCH", default_value = "x86_64")]
     arch: String,
+
+    /// The variant to build.
+    #[clap(env = "BUILDSYS_VARIANT")]
+    variant: String,
+
+    /// The go modules that should be build
+    #[clap(long = "go-modules", env = "GO_MODULES", default_value = "")]
+    go_modules: String,
 }
 
 impl BuildVariant {
     pub(super) async fn run(&self) -> Result<()> {
-        Ok(())
+        let project = project::load_or_find_project(self.project_path.clone()).await?;
+        let tempdir = tools_tempdir()?;
+        install_tools(&tempdir).await?;
+        let makefile_path = tempdir.path().join("Makefile.toml");
+        let packages_dir =
+            TempDir::new().context("Unable to create a tempdir for Twoliter's packages")?;
+
+        let sdk_container = DockerContainer::new(
+            "sdk",
+            project
+                .sdk(&self.arch)
+                .context("The project was missing an sdk")?
+                .uri(),
+        )
+        .await?;
+        sdk_container
+            .cp(Path::new("twoliter/alpha/build/rpms"), packages_dir.path())
+            .await?;
```

Comment:
I would rather we modify the way we're building the alpha SDK to add repodata to the local path where we're copying the RPMs. Then adjust the `dnf` install lines in the `Dockerfile` to enable the "alpha" repo.

Ultimately kits will work something like this - RPMs copied out of containers along with repo definitions, to be used automatically when `dnf` runs.